stable/moodle/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
 - name: mariadb
-  version: 0.7.0
+  version: 3.x.x
```

Comment:
Locking to a major version series feels like a very risky move. At best we should lock down to a minor version. 
Having said that I think we no longer suggest locking to version series. I can't remember the exact reason /cc @prydonius

Reply:
There shouldn't be any new MariaDB release matching 3.x.x as we are already in the 4.x.x version.
I tested the chart with this specific version of MariaDB and it works fine